### DISCUSSION TITLE
Gauge Test Rail Integration

### DISCUSSION THREAD

**Discussion by jayamvishnudeep** [getgauge/gauge#2063] [General]
Hello,

I wanted to know if we can Integrate Gauge Framework (used Python) with Test Rail?

I want it to Import test results into TestRail via API. Are there any implementation examples?

**Comment by sriv** [comment]
@ajoecker has written a plugin for gauge+testrail integration: https://github.com/ajoecker/gauge-testrail

**Reply by edgar-chavez-bdev** [reply]
That repo has a link to malware home page.

**Reply by chadlwilson** [reply]
Please report to that project repo. We can't do anything about it here.

Unfortunately the old Gauge domain was allowed to expire and now it's being squatted on.
